client/app/styles/main.less

```diff
@@ -187,11 +188,14 @@ h2 {
 }
 
 .footer {
+  padding: @footer-padding;
+  margin: -@footer-padding;
```

Comment:
Maybe just adjust the `bottom` and `right` properties below instead of using negative margins. A little easier to read/comprehend.